app/controllers/root_controller.rb

```diff
@@ -0,0 +1,3 @@
+class RootController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] I can't see any route that points to this controller. Please consider adding it 🤓 🔧

Reply:
Resolved